cpp/src/gandiva/tests/projector_test.cc

```diff
@@ -65,14 +71,17 @@ TEST_F(TestProjector, TestProjectCache) {
   std::shared_ptr<Projector> projector;
   auto status = Projector::Make(schema, {sum_expr, sub_expr}, configuration, &projector);
   ASSERT_OK(status);
+  EXPECT_FALSE(projector->GetCompiledFromCache());
 
   // everything is same, should return the same projector.
   auto schema_same = arrow::schema({field0, field1});
   std::shared_ptr<Projector> cached_projector;
   status = Projector::Make(schema_same, {sum_expr, sub_expr}, configuration,
                            &cached_projector);
+
   ASSERT_OK(status);
-  EXPECT_EQ(cached_projector, projector);
+  //EXPECT_EQ(cached_projector, projector); //-> old expect.
```

Comment:
Remove the comment if it is not necessary

Reply:
Removed, it is unnecessary.